src/charm.py

```diff
@@ -92,8 +103,7 @@ def _create_service(self):
             refresh_event=self.on.config_changed,
         )
 
-    @staticmethod
-    def _get_env_vars(relational_db_data, object_storage_data):
+    def _get_env_vars(self, relational_db_data, object_storage_data):
         """Return environment variables based on model configuration."""
```

Comment:
We started to covert this to property `service_environment`. Just a suggestion.

Reply:
To keep the pr small with just the context of the task I propose openning issue and facing it in another pr.

Reply:
@misohu 
you probably misunderstood

you just update method name and add `@property`

Reply:
not that easy ... I need to create setter ... and fix 4 tests ... I will do it in PR for get_password action ok ? :)

Reply:
you do not need setter

Reply:
you will need to put relational_db_data, object_storage_data into the property itself, rather than pointing from outside

Reply:
talked with Michal, too much refactoring for the current PR. Handle it in another PR

Reply:
Sounds goof.